augur/curate/apply_geolocation_rules.py

```diff
@@ -9,6 +9,10 @@
 from augur.version import __version__
 
 
+class NoGeolocationRulesProvidedError(AugurError):
+    pass
```

Comment:
[not a request for changes] I'm curious, what's the benefit of a new error vs `AugurError`?